### PR TITLE
#1419 Implement citation lookup in Python interface for new test

### PR DESCRIPTION
**Pull request by coolalexzb**
Hi, I merged commits in #1428 in this pr.

### CODE REVIEW

File: retriever/__main__.py [hunk 1]
```diff
@@ -62,16 +62,15 @@ def main():
...
                 print(CITATION)
             else:
-                scripts = name_matches(script_list, args.dataset)
-                for dataset in scripts:
-                    print("\nDataset:  {}".format(dataset.name))
-                    print("Citation:   {}".format(dataset.citation))
-                    print("Description:   {}\n".format(dataset.description))
-
+                for citation in citations:
+                    print("Citation:   {}".format(citation))
```
Comment (apoorvaeternity) [inline retriever/__main__.py]: Moving forward, I think we can start making use of python `f-strings`. https://www.python.org/dev/peps/pep-0498/
The above statement can be written as:
```python
print(f"Citation:   {citation}")
```

Comment (coolalexzb) [reply retriever/__main__.py]: Thanks for your guide. I found that many expression in __main__.py is like  `print("Citation:   {}".format(citation))`. Do I need to change all of the expression to the example you provided? Thank you!

Comment (henrykironde) [reply retriever/__main__.py]: Not for this PR. you can change those others  print statements in a different PR.
Let this PR have only the related changes to the issue. i.e citation related

Comment (coolalexzb) [reply retriever/__main__.py]: Ok, I will modify the code.

Comment (coolalexzb) [reply retriever/__main__.py]: Before I pull request, I want to know how can I only use AppVeyor test like #1434 
Thank you！

Comment (henrykironde) [reply retriever/__main__.py]: You can use only Appveyor for test. I am trying to make tests on linux work again

Comment (coolalexzb) [reply retriever/__main__.py]: Does Travis CI build work now? Is there any approach to test the code in this pr again without additional commit and push?

Comment (henrykironde) [reply retriever/__main__.py]: Have you tried the Docker setup. We have docs on testing using docker

Comment (coolalexzb) [reply retriever/__main__.py]: Ok, I will try to use docker. Thanks!

**Comment by henrykironde** [comment]
Thanks  @coolalexzb for contributing to this issue

**Pull request closed without merging**